### ISSUE TITLE
Push not sent for iOS

### ISSUE THREAD

**Issue by recsouza1** [onmyway133/PushNotifications#48]
Hey guys, trying to send pushes for iOS but I see a forever loading after clicking on Send button. It used to work fine before.

Thank you

**Comment by Ariandr** [comment]
Hi @recsouza1 
Do you use the latest app version?

If not, you might have moved or removed the file at the previous location (certificate or token). Please, check.

**Comment by Ariandr** [comment]
That's probably the issue you have: https://github.com/onmyway133/PushNotifications/issues/28#issuecomment-705453721

**Comment by recsouza1** [comment]
Hello, yes, I have version 1.7.7 for Windows installed.

**Comment by Ariandr** [comment]
@recsouza1 
In what version it worked fine for you, by the way?

**Comment by Ariandr** [comment]
@recsouza1 
I added `.msi` for Windows of app v.1.8.0
Please, try to install it and reselect the `.pm` file.
https://github.com/onmyway133/PushNotifications/releases/tag/1.8.0

**Comment by recsouza1** [comment]
@Ariandr tried here but it is still loading forever, treid a different .p12 file too

**Comment by recsouza1** [comment]
@Ariandr now it is not even letting me copy / past the json in the app, it only pastes the {}

**Comment by Ariandr** [comment]
@recsouza1 
Well, it works for me on windows.

**Comment by Ariandr** [comment]
@recsouza1 
And copy-pasting too, just after pasting you need to click away

**Comment by Ariandr** [comment]
@recsouza1 
You are right, I found some issues with that JSON editor. I deleted the release for now.

**Comment by Ariandr** [comment]
@Ariandr 
But regarding the initial issue, I don't know what might be wrong in your case.

**Comment by recsouza1** [comment]
Thank you, It hasn't been working for iOS since, I have no idea why too :(

**Comment by zykis** [comment]
same issue here

**Comment by CoBug92** [comment]
same
